ui/lib/apps/SlowQuery/components/List.tsx

```diff
@@ -172,10 +178,7 @@ export default function List() {
                 </Option>
               ))}
             </Select>
-            <Search
-              value={searchText}
-              onChange={(e) => setSearchText(e.target.value)}
-            />
+            <Search onSearch={handleSearch} />
```

Comment:
I did this before, but this way can't recover the search text when comes back from detail page.

Reply:
Now we store all params in router query, this issue resolved, PTAL